pyInterface/bindings/partialWaveFit/fitResult_py.cc

```diff
@@ -60,14 +60,53 @@ namespace {
 			PyErr_SetString(PyExc_TypeError, "Got invalid input for phaseSpaceIntegral when executing rpwa::fitResult::fill()");
 			bp::throw_error_already_set();
 		}
-		self.fill(nmbEvents, normNmbEvents, massBinCenter, logLikelihood, rank, prodAmps, prodAmpNames, *fitParCovMatrix,
+		std::map<std::string, std::pair<double, double> > cBinningMap;
+		const bp::list keys = binningMap.keys();
+		for (int i = 0; i < bp::len(keys); ++i) {
+			std::string binningVar = bp::extract<std::string>(keys[i]);
+			double lowerBound      = bp::extract<double>(binningMap[binningVar][0]);
+			double upperBound      = bp::extract<double>(binningMap[binningVar][1]);
+			cBinningMap[binningVar] = std::pair<double, double>(lowerBound, upperBound);
+		}
+		self.fill(nmbEvents, normNmbEvents, cBinningMap, logLikelihood, rank, prodAmps, prodAmpNames, *fitParCovMatrix,
 		          fitParCovMatrixIndices, normIntegral, acceptedNormIntegral, phaseSpaceIntegral, converged, hasHessian);
 	}
 
 	void fitResult_fill_2(rpwa::fitResult& self, const rpwa::fitResult& result) {
 		self.fill(result);
 	}
 
+	void fitResult_setBinningMap(rpwa::fitResult& self, const bp::dict& binningMap) {
+		std::map<std::string, std::pair<double, double> > cBinningMap;
+		const bp::list keys = binningMap.keys();
+		for (int i = 0; i < bp::len(keys); ++i) {
+			std::string binningVar = bp::extract<std::string>(keys[i]);
+			double lowerBound      = bp::extract<double>(binningMap[binningVar][0]);
+			double upperBound      = bp::extract<double>(binningMap[binningVar][1]);
+			cBinningMap[binningVar] = std::pair<double, double>(lowerBound, upperBound);
+		}
+		self.setBinningMap(cBinningMap);
+	}
```

Comment:
given that the `setBinningMap` function should only be called by the ROOT streamer, this function should probably not be exported to Python.